packages/core/src/plugin/default-search-plugin/default-search-plugin.ts

```diff
@@ -192,6 +195,16 @@ export class DefaultSearchPlugin implements OnApplicationBootstrap {
                     return this.searchIndexService.reindex(event.ctx);
                 }
             });
+
+        await this.initSearchStrategy();
+    }
+
+    private async initSearchStrategy(): Promise<void> {
+        const injector = new Injector(this.moduleRef);
+        const searchService = injector.get(FulltextSearchService);
+        if (typeof searchService.searchStrategy.init === 'function') {
+            await searchService.searchStrategy.init(injector);
+        }
     }
 
```

Comment:
We should also have a corresponding `destroySearchStrategy()` method here, in case the strategy implements the `destroy()` method. It would be called from the `onApplicationShutdown` hook (see the config module for example)

Reply:
Done 👍